src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let memory = [];
+
+  const adder = (number) => {
```

Comment:
The parameter name 'number' could be misleading since it's not always a number. Consider renaming it to something more generic like 'value' to reflect that it can also be 'undefined'.